components/molecules/Greeting.js

```diff
@@ -5,10 +5,25 @@ import fr from '../../locales/fr'
 export default function Greeting(props) {
   const t = props.locale === 'en' ? en : fr
 
+  function getGreeting() {
+    switch (true) {
+      case props.time < 12:
```

Comment:
You could add another case for `props.time < 6` if a user happens to be burning the midnight oil  
```suggestion
      case props.time < 6:
        return t.eveningGreeting
      case props.time < 12:
```